Code/SimDivPickers/MaxMinPicker.h

```diff
@@ -133,71 +141,115 @@ RDKit::INT_VECT MaxMinPicker::lazyPick(T &func, unsigned int poolSize,
   if (poolSize < pickSize)
```

Comment:
poolSize needs to be > 0, I can't see if that is checked.

Perhaps make both PRECONDITIONS